### ISSUE TITLE
[Effect Suggestion] Spawn Mechanic

### ISSUE THREAD

**Issue by DEATHCON1** [gta-chaos-mod/ChaosModV#1160]
**Name of effect**
Spawn Mechanic
Spawn Online Mechanic

**Effect Description**
Spawns the mechanic from Online that steals and deives off with the player's last used or current vehicle.

**Comment by Thunderstorm64** [comment]
NO more spawn this or that.

**Comment by ProfessorBiddle** [comment]
> NO more spawn this or that.

not necessarily, if a 'spawn' suggestion is unique and interesting we'll implement it. This one is similar to a lot of other effects/suggestions.

**Comment by Last0xygen** [comment]
Sounds a lot like https://github.com/gta-chaos-mod/ChaosModV/pull/524

**Comment by DEATHCON1** [comment]
> Sounds a lot like #524

Well, instead of wanting to take your money, this guy will want to take your vehicle. Like he did before on PS3 and XBOX 360 online mode, before they removed him...
